modules/sql/Log.js

```diff
@@ -1,20 +1,31 @@
-const Database = require('better-sqlite3');
+const Database = require("better-sqlite3");
 const { join } = require("path");
 
 class Log {
     constructor(path) {
-        this.db = new Database(join(path, "system.db")/*, { verbose: console.log }*/);
+        this.db = new Database(join(path, "system.db"));
 
-        this.db.exec("CREATE TABLE IF NOT EXISTS logs(type TEXT, action TEXT, message TEXT, time TEXT)");
```

Comment:
not a grip on your code, your just following the example we setout, but we should really consider implementing a dedicated logger like Winston> Then through the use of a plugin, we collect the data we need from the logger.